### ISSUE TITLE
Darker checkbox border

### ISSUE THREAD

**Issue by ainemckay** [MHRA/products#927]
While the default (un-modified) user agent checkbox styling does not fail accessibility checks, for better accessibility and
consistency with other parts of site they should be updated.

Recommend styling checkbox to give it a darker border – see Disclaimer checkbox example provided

## Acceptance criteria

- [x] Filter checkboxes have dark border

## Testing instructions
- Go to non-prod site
- Make a search
- View checkboxes and confirm border is darker compared to production site

**Comment by TimboTambo** [comment]
QA report: checkbox appears darker and more obvious, in line with recommendations.
